src/main/java/com/webservices/company/repository/CompanyRepository.java

```diff
@@ -48,7 +49,12 @@ public Company add(Company companyToBeAdded) {
 
     @Override
     public Company get(Long id) {
-        Company company = jdbcTemplate.queryForObject("select * from companies where id = " + id, rowMapper);
+        Company company = null;
+        try {
+            company = jdbcTemplate.queryForObject("select * from companies where id =  " + id, rowMapper);
+        } catch (ResourceNotFoundException e){
```

Comment:
in what case ResourceNotFoundException is thrown? com.webservices.company.exceptions.ResourceNotFoundException is your custom exception, jdbcTemplate.queryForObject will never throw this exception